v4l2/videodev.py

```diff
@@ -324,7 +324,10 @@ def set_format(self):
         assert pix.pixelformat == self.format.v4l2_fourcc, f'{pix.pixelformat} != {self.format.v4l2_fourcc}'
         assert pix.width == self.width
         assert pix.height == self.height
-        assert pix.bytesperline == self.strides[0], f'{pix.bytesperline} != {self.strides[0]}'
+        # TODO: find better workaround
+        if pix.bytesperline != self.strides[0]:
+            print(f'WARNING - stride issue: {pix.bytesperline} != {self.strides[0]}')
+            self.strides[0] = pix.bytesperline
```

Comment:
You need to describe the problem and how the commit fixes it.

I know strides are not well supported at the moment. I'd rather see that implemented, than adding workarounds.

Reply:
Yes sure ! Please drop this commit, I need to investigate the root cause ... 
It was just a workaround I had to make it work.